res/css/views/elements/_CopyableText.scss

```diff
@@ -22,6 +22,7 @@ limitations under the License.
     max-width: 100%;
 
     &.mx_CopyableText_border {
+        width: 100%;
```

Comment:
This will apply to all copyable text in the whole app. I don't think that is what we want - we want just this part to be affected.

Reply:
Alright. I understand